contexts/tz-2021-v2.jsonld

```diff
@@ -0,0 +1,106 @@
+{
```

Comment:
Could we please make issues to track each signature scheme we need to formalize into a spec? It can be a single issue with a checklist.

Reply:
Opened: https://github.com/spruceid/ssi/issues/116